rachellegelden/Home-Together-Canada
# HTC-735: route to give admin privileges

# [HTC-735](https://github.com/rachellegelden/Home-Together-Canada/issues/735)

## Summary
- added a route that allows an admin to give admin privileges to another member

## Relevant Motivation & Context
- will be used in the admin problem

## Testing Instructions
1. create a member
2. while authenticated on Postman, ping `http://localhost:3001/admin/dev/create/` with a `GET` request to make that user an admin
3. ping `http://localhost:3001/admin/create/` with a `POST` request and the following request body to make the provided user an admin:
```
{
    "username": "here_to_PARTY"
}
```
4. login with that member you just gave admin privileges to on the website. Make sure the Admin button is shown in the header
## Developer checklist prior to opening this pull request:


## Reviewer
